AutomatedTesting/Gem/PythonTests/AWS/README.md

```diff
@@ -20,17 +24,19 @@
 3. Open a new CLI window at the engine root and set the following environment variables:
     * Windows
         ```
-            Set O3DE_AWS_PROJECT_NAME=AWSAUTO
-            Set O3DE_AWS_DEPLOY_REGION=us-east-1
-            Set ASSUME_ROLE_ARN=arn:aws:iam::{your_aws_account_id}:role/o3de-automation-tests
-            Set COMMIT_ID=HEAD
+            set O3DE_AWS_PROJECT_NAME=AWSAUTO
+            set O3DE_AWS_DEPLOY_REGION=us-east-1
+            set ASSUME_ROLE_ARN=arn:aws:iam::{your_aws_account_id}:role/o3de-automation-tests
+            set COMMIT_ID=HEAD
+            set CDK_VERSION=2.39.1

```

Comment:
will the tests fail on another version of the CDK v2?

Reply:
No, was unclear how to provide an example, this is latest version

Reply:
what happens if this isn't set at all, will it attempt to use the latest CDK v2? I understand for automation testing it's useful to be explicit about what version to use but requiring it adds a bit more friction for local users